File: 백준/Bronze/27475. Cancel the Trains/Cancel the Trains.py
import sys
input=lambda:sys.stdin.readline().rstrip()
t=int(input())
for _ in range(t):
    n,m=map(int,input().split())
    x=[*map(int,input().split())]
    y=[*map(int,input().split())]
    cnt=0
    for i in y:
        for j in x:
            for t in range(101):
                if i==t and t==j:
                    cnt+=1
    print(cnt)

File: 백준/Bronze/27475. Cancel the Trains/Cancel the Trains.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int t, n, m, cnt;
    cin >> t;
    for (int l = 0; l < t; l++) {
        cin >> n >> m;
        cnt = 0;
        vector<int> x(n), y(m);
        for (int i = 0; i < n; i++)
            cin >> x[i];
        for (int j = 0; j < m; j++)
            cin >> y[j];
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < m; j++) {
                for (int k = 0; k < 101; k++) {
                    if (x[i] == k && k == y[j]) {
                        cnt += 1;
                    }
                }
            }
        }
        cout << cnt << "\n";
    }
    return 0;
}